src/main/java/com/exadel/movie_new/controller/MovieController.java

```diff
@@ -1,48 +1,47 @@
 package com.exadel.movie_new.controller;
 
+import com.exadel.movie_new.exception.MovieNotFoundException;
 import com.exadel.movie_new.model.Movie;
 import com.exadel.movie_new.service.MovieService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
-
-@RestController @RequestMapping("/movies")
+@RestController
+@RequestMapping("/movies")
 public class MovieController {
 
-    private MovieService movieService;
+    private final MovieService movieService;
 
     public MovieController(MovieService movieService) {
         this.movieService = movieService;
     }
 
-
-    @PostMapping()
-    public Movie addMovie(@RequestBody Movie movie) {
-
-        return movieService.addMovie(movie);
+    @PostMapping
+    public ResponseEntity<Movie> addMovie(@RequestBody Movie movie) {
+        return new ResponseEntity<>(movieService.addMovie(movie), HttpStatus.CREATED);
     }
 
     @GetMapping("/{movieId}")
-    public Movie getMovie(@PathVariable String movieId) {
-
-        return movieService.getMovie(movieId);
+    public ResponseEntity<Movie> getMovie(@PathVariable String movieId) {
+        return new ResponseEntity<>(movieService.getMovie(movieId), HttpStatus.OK);
     }
 
-    @GetMapping()
-    public List<Movie> getAllMovies() {
-
-        return movieService.getAllMovies();
+    @GetMapping
+    public ResponseEntity<List<Movie>> getAllMovies() {
+        return new ResponseEntity<>(movieService.getAllMovies(), HttpStatus.OK);
     }
-    
-    @PutMapping("/{movieId}")
-    public Movie updateMovie(@RequestBody Movie movie) {
-        return movieService.updateMovie(movie);
+
+    @PutMapping
+    public ResponseEntity<Movie> updateMovie(@RequestBody Movie movie) {
+        return new ResponseEntity<>(movieService.updateMovie(movie), HttpStatus.CREATED);

```

Comment:
I think Accepted would be more preferable, because we do not create movie but just update existing one.